types/vote_set.go

```diff
@@ -541,11 +541,13 @@ func (voteSet *VoteSet) MakeCommit() *Commit {
 	}
 
 	// For every validator, get the precommit
-	votesCopy := make([]*Vote, len(voteSet.votes))
-	copy(votesCopy, voteSet.votes)
+	commitSigs := make([]*CommitSig, len(voteSet.votes))
```

Comment:
On the second thought I think we should name it `precommits` since that's what we're building and later use to construct a Commit.

Reply:
Hmm. I don't think either name is all the great to be honest. Better might be `PrecommitWitness`. A precommit is really the whole Vote structure with the PrecommitType. I'll comment on #1648 and we can address the renaming as we go.